tests/functional/storageclass/test_rbd_default_storageclass.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import logging
+from ocs_ci.helpers.helpers import is_rbd_default_storage_class
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.helpers.helpers import default_storage_class
+from ocs_ci.framework.pytest_customization.marks import tier1, green_squad, polarion_id
+
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
+class TestRBDStorageClassAsDefaultStorageClass:
+    @tier1
+    @polarion_id("OCS-5459")
+    @pytest.mark.skipif(
+        not is_rbd_default_storage_class(),
```

Comment:
You are running a test in the marker itself. We should check if it is (BM or Vsphere) and UI. Then in the test  run the check. Beside that it fails on collectony see PR38 checks. If we really wanted to this the way is to create a new mark and in the mark you need to put a condition for (BM or Vsphere) and UI and kubeconfig defined and only if this condition work we run `is_rbd_default_storage_class` . This way in the collectonly you'll not fail.